portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase

```diff
@@ -32,14 +32,42 @@ definition {
 	}
 
 	@description = "Verify that an Image Field can be duplicated"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeDuplicated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133033 CanBeDuplicated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Image");
+
+		FormViewBuilder.editText(
+			fieldName = "Help Text",
+			fieldValue = "Help Text Test");
+
+		FormFields.duplicateField();
+
+		FormViewBuilder.validateFieldLabel(fieldLabel = "Image");
+
+		FormViewBuilder.validateFieldLabel(fieldLabel = "Copy of Image");
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Image",
+			fieldValue = "Help Text Test");
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Copy of Image",
```

Comment:
@leticiachiu, shouldn't we use `fieldLabel` instead of `fieldName`?

After all, **Image** and **Copy of Image** are the fields label and not the fields name. :slightly_smiling_face:

Reply:
Hey @leticiachiu, recently, Forms Team detected that the macro `FormViewBuilder` is an external resource from other product (App Builder) and, ideally, we shouldn't be using it on Forms Tests. However, once Forms Team just realized this issue now, let's keep using it for now. Soon, I'll get in touch with all interns regarding the next steps related to this matter. :slightly_smiling_face: 

Can you please put back the `fieldName` instead of `fieldLabel` on the macro `assertHelpTextPresent`? It's being used not only on Forms, consequently this pull can affect the Poshi tests of other products, like Data Engine, for example. Sorry to give you this extra work, @leticiachiu. :grin: